src/components/duration/normalized.rs

```diff
@@ -118,21 +145,636 @@ impl Add<Self> for NormalizedTimeDuration {
     }
 }
 
-/// A normalized `DurationRecord` that contains a `DateDuration` and `NormalizedTimeDuration`.
+// ==== NormalizedDurationRecord ====
+//
+// A record consisting of a DateDuration and NormalizedTimeDuration
+//
+
+/// A NormalizedDurationRecord is a duration record that contains
+/// a `DateDuration` and `NormalizedTimeDuration`.
 #[derive(Debug, Clone, Copy)]
 pub struct NormalizedDurationRecord(pub(crate) (DateDuration, NormalizedTimeDuration));
```

Comment:
I think it would be better to use a proper struct instead of a tuple here. Also, can we make this private? I'm not sure if we need to allow external mutations here, seeing that we require checks on `new` that may not be preserved by external code.